salt/crypt.py

```diff
@@ -330,100 +327,35 @@ def decrypt(self, data):
 
 
 @salt.utils.decorators.memoize
-def _get_key_with_evict(path, timestamp, passphrase):
-    """
-    Load a private key from disk.  `timestamp` above is intended to be the
-    timestamp of the file's last modification. This fn is memoized so if it is
-    called with the same path and timestamp (the file's last modified time) the
-    second time the result is returned from the memoization.  If the file gets
-    modified then the params are different and the key is loaded from disk.
-    """
-    log.debug("salt.crypt._get_key_with_evict: Loading private key")
-    if passphrase:
-        password = passphrase.encode()
-    else:
-        password = None
-    with salt.utils.files.fopen(path, "rb") as f:
-        return serialization.load_pem_private_key(
-            f.read(),
-            password=password,
-        )
-
-
 def get_rsa_key(path, passphrase):
```

Comment:
the memoize is still here, but get_rsa_key is no longer used when loading the master keys. i dont think this should be a big deal, but calling it out